addons/account/models/account_bank_statement_line.py

```diff
@@ -455,14 +455,19 @@ def _find_or_create_bank_account(self):
         return bank_account.filtered(lambda x: x.company_id in (False, self.company_id))
 
     def _get_default_amls_matching_domain(self):
+        all_reconcilable_account_ids = self.env['account.account'].search([
```

Comment:
can you add a `self.ensure_one()` plz?

Reply:
So it doesn't work if we inject the subquery, we have to execute it beforehands?

Reply:
> So it doesn't work if we inject the subquery, we have to execute it beforehands?

No, because this domain is given to the view